src/Framework/Config/GacelaFileConfig/Factory/GacelaConfigUsingGacelaPhpFileFactory.php

```diff
@@ -49,15 +49,34 @@ public function createGacelaFileConfig(): GacelaConfigFile
             throw new RuntimeException('Your anonymous class must extends AbstractConfigGacela');
         }
 
+        $configBuilder = $this->createConfigBuilder($configGacelaClass);
+        $mappingInterfacesBuilder = $this->createMappingInterfacesBuilder($configGacelaClass);
+        $suffixTypesBuilder = $this->createSuffixTypesBuilder($configGacelaClass);
+
+        return GacelaConfigFile::usingBuilders($configBuilder, $mappingInterfacesBuilder, $suffixTypesBuilder);
```

Comment:
Just another alternative to think about, nor better or worse
```php
return GacelaConfigFile::usingBuilders(
    $this->createConfigBuilder($configGacelaClass),
    $this->createMappingInterfacesBuilder($configGacelaClass),
    $this->createSuffixTypesBuilder($configGacelaClass)
);
```

Reply:
I thought about this too, but my first impression is that it's "too much compress"... maybe we'll change our mind and refactor it in the future, but for now I would prefer keep it with the variables :)